tox.ini

```diff
@@ -53,13 +53,13 @@ commands =
 deps =
     -r{toxinidir}/requirements/dev.txt
 commands =
-    isort --recursive tests test_utils enterprise enterprise_learner_portal consent integrated_channels manage.py setup.py
+    isort --skip migrations --recursive tests test_utils enterprise enterprise_learner_portal consent integrated_channels manage.py setup.py
```

Comment:
same question about isort skipping migrations; just to be sure we're OK with that even when a migration file may not always be auto-generated.